### ISSUE TITLE
Custom themes don't work

### ISSUE THREAD

**Issue by gespinha** [ag-grid/ag-grid#8362]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Customers of AG Grid**

If you are a customer you are entitled to use the AG Grid's customer support system (powered by Zendesk). Please use that channel for guaranteed response from the AG Grid team with regards bugs, feature requests and support.

**Requesting Community Support**

If you are not a customer of AG Grid, ag-grid staff will label your issue as managed-by-the-community. This means that AG Grid staff is not going to be actively looking into it and it will get closed if inactive for more than one month. The community is welcome to help with this question/support issue.

**Providing a Reproducible Scenario**
Accepted reproducible scenarios are
- A description of the detailed steps to reproduce your behaviour in one of our examples in the docs.
- A plunker

If you decide to send us a plunker, from any example in our website use the plunker button in there to fork your own code by following the steps below:

- Select the framework that is appropriate to you from the drop-down
- Open it in plunker. (Use the button plunker in our example)
- Add your changes so that the behaviour is reproduced
- Save and Freeze the plunker(On the top left corner)
- Send us the link to the plunker(You can copy the URL from the browser)

If reporting a bug make sure to state.

Current behaviour.
Expected behaviour. If possible back this up with our docs/examples if possible

**Current behavior**
- Create a custom theme in the Theme Builder page on the website
- Download the theme file and add it to my Vue project
- Add a 'ag-theme-custom' class to the <AgGrid> component as described in the documentation guide
- Grid appears to work with some styles, but stuff like the cell height is not rendering correctly despite the values being correct in the downloaded save file

**Expected behavior**
- The grid renders according to the edited fields in the Theme Builder

**Please tell us about your environment:**
VueJs app running in a Docker container

* **AG Grid version:** 31.0.1

* **Browser:** 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 Edg/126.0.0.0'

* **Language:** [ES5]

**Comment by bbeas** [comment]
same problem with AGGridReact. AG Grid version: 31.2.1
The css file downloaded from theme-builder seems incorrect. Following the instructions here doesn't work. https://www.ag-grid.com/react-data-grid/applying-theme-builder-styling-grid/

**Comment by AG-Zoheil** [comment]
Hi,

Thank you for reporting this.

We'd be happy to look into this, but the most efficient way for us to do so would be for you to provide a live example reproducing the issue.

Please send us a live plunker sample which shows the issue. You can use one of the examples from our website as a starting point. Do let us know if you need further guidance on this.

We're looking forward to your response.

Kind regards,
Zoheil

**Comment by gespinha** [comment]
> Hi,
> 
> Thank you for reporting this.
> 
> We'd be happy to look into this, but the most efficient way for us to do so would be for you to provide a live example reproducing the issue.
> 
> Please send us a live plunker sample which shows the issue. You can use one of the examples from our website as a starting point. Do let us know if you need further guidance on this.
> 
> We're looking forward to your response.
> 
> Kind regards, Zoheil

Sure! I've created a working [example here](https://codesandbox.io/p/devbox/ag-grid-theme-test-dm5lpw?file=%2Fsrc%2Fcomponents%2FAgGrid.vue%3A7%2C1&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clyr5fkek00073p6j1cdivg71%2522%252C%2522sizes%2522%253A%255B60%252C40%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clyr5fkek00023p6jadzabfh0%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clyr5fkek00043p6jnjd4nfk2%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clyr5fkek00063p6jyj0ztkcl%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B60.644760059008995%252C39.355239940991005%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clyr5fkek00023p6jadzabfh0%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clyr5fkek00013p6j3kunu8mr%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522clyr5sxv000023p6kqzhvblqm%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A7%252C%2522startColumn%2522%253A1%252C%2522endLineNumber%2522%253A7%252C%2522endColumn%2522%253A1%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252Fcomponents%252FAgGrid.vue%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522clyr5fkek00023p6jadzabfh0%2522%252C%2522activeTabId%2522%253A%2522clyr5sxv000023p6kqzhvblqm%2522%257D%252C%2522clyr5fkek00063p6jyj0ztkcl%2522%253A%257B%2522id%2522%253A%2522clyr5fkek00063p6jyj0ztkcl%2522%252C%2522activeTabId%2522%253A%2522clyr5uya100bt3p6kfavqv4vk%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clyr5fkek00053p6jrd30us4s%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522pnpm%2520run%2520dev%2522%252C%2522port%2522%253A5173%252C%2522path%2522%253A%2522%252F%2522%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A5173%252C%2522id%2522%253A%2522clyr5uya100bt3p6kfavqv4vk%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clyr5fkek00043p6jnjd4nfk2%2522%253A%257B%2522id%2522%253A%2522clyr5fkek00043p6jnjd4nfk2%2522%252C%2522activeTabId%2522%253A%2522clyr5ulzy009e3p6kn41ew2gp%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clyr5jm7a00co3p6jbsnoroi7%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clyr5jm9b0079dbhugyp878lx%2522%257D%252C%257B%2522id%2522%253A%2522clyr5ulzy009e3p6kn41ew2gp%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clyr5um1k007jdbhuf4qdfv0l%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

In that example, you'll see a screenshot of what I created in Theme Builder and below a working example of what I got by following [this guide](https://www.ag-grid.com/javascript-data-grid/applying-theme-builder-styling-grid/).

**Comment by BernieSumption** [comment]
Hi,

The issue is that your downloaded theme is for v32.0 and your grid is on v31.0. For the moment (we're currently working on lifting this restriction) theme builder downloads can only be used with the current version of the grid.

You can either update your grid to 32.0, or use an older version of Theme Builder. You can access old versions of Theme Builder in our archive, e.g. https://ag-grid.com/archive/31.3.0/theme-builder/

Theme Builder was released with version 31.3 so in any case you'd need to update from 31.0 to 31.3.
